### DISCUSSION TITLE
Problem mit Courseplay will sich immer reparieren

### DISCUSSION THREAD

**Discussion by benne43** [Courseplay/Courseplay_FS22#2574] [Q&A]
Hallo ich habe ein problem mit courseplay denn wenn ich ein kurs generiere und starte steht da dann Muss repariert werden und das immer wieder. habe log hinterlegt
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/11635399/log.txt)

**Comment by Tensuko** [comment]
Glaubst du wirklich wir lassen uns hier verkaspern?
Nicht nur das du die Discussion jetzt schon zum dritten mal auf machst, nein du benutzt auch noch einen neuen Nickname und postest 1:1 die gleiche log.
Wir bekommen immer eine Mail wenn jemand etwas schreibt und ein "schreibt hier noch jemand?" etc oder auf zu auf zu führt ganz bestimmt NICHT dazu, dass IRGENDJEMAND Lust dazu hat sich dein Problem weiter an zu gucken!

Du hast ein Problem? Ok melde dich und wir helfen dir WENN wir können und WANN wir können, aber nicht so!
Entweder du wartest es jetzt ab, bis einer von uns die Zeit gefunden hat, sich dein Problem genauer an zu gucken, oder ich entferne deine Anfrage, denn ich bin mitrzu 100% sicher, an CP liegt es nicht, sonst hätten mehrere das Problem, ich inklusive.

Original Discussion: https://github.com/Courseplay/Courseplay_FS22/discussions/2561
